src/components/MapContainer.jsx

```diff
@@ -1,18 +1,94 @@
 import React from 'react';
-import { Map, GoogleApiWrapper, Marker } from 'google-maps-react';
-
-const MapContainer = ({ google }) => {
-  return (
-    <Map
-      google={google}
-      zoom={5}
-      initialCenter={{ lat: 19.5943885, lng: -97.9526044 }}
-    >
-      <Marker
-        position={{ lat: 19.4267261, lng: -99.1718706 }}
-      />
-    </Map>
-  );
+import { Map, GoogleApiWrapper, Marker, InfoWindow } from 'google-maps-react';
+
+
+class MapContainer extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = { 
+      show: false,
+      showingInfoWindow: false,
+      activeMarker: {},
+      selectedPlace: {},
+       }
+  }
+
+  handleClick = () => {
+    const { show } = this.state;
```

Comment:
Es mejor utilizar esta forma de hacer el if, para el manejo de estados.

```suggestion
 show: !this.state.show,
```